### PR TITLE
Format entire package during render and eval

### PR DESCRIPTION
**Pull request by phanimarupaka**
This PR is to add the formatting as postRun step for eval and render commands. Invoking any of these commands will ensure consistent formatting of all resources(including meta-resources) in the package.

Most of the diffs in the PR are on formatting and are repetitive.

### CODE REVIEW

**Comment by droot** [comment]
@phanimarupaka looks like the output from `enforce-gatekeeper` is not stable that is causing the test to be flaky. @Shell32-Natsu or @mengqiy will skip the test till the fix lands in `enforce-gatekeeper`.

File: thirdparty/cmdconfig/commands/cmdeval/cmdeval.go [hunk 1]
```diff
@@ -293,3 +297,29 @@ func (r *EvalFnRunner) preRunE(c *cobra.Command, args []string) error {
...
+	}
+	err := kio.Pipeline{
+		Inputs:  []kio.Reader{inout},
+		Filters: []kio.Filter{f},
+		Outputs: []kio.Writer{inout},
+	}.Execute()
+	if err != nil {
+		// do not throw error if formatting fails
+		return
+	}
```
Comment (droot) [inline thirdparty/cmdconfig/commands/cmdeval/cmdeval.go:324]: Looks like we can have a method can be moved under `pkgutils` or `pkg` and can be invoked from the postRuns of both `eval` and `render`

**Review by droot — CHANGES_REQUESTED** [review]

File: thirdparty/cmdconfig/commands/cmdeval/cmdeval.go (hunk 1 above)
Comment (phanimarupaka) [reply thirdparty/cmdconfig/commands/cmdeval/cmdeval.go]: @droot Updated.

**Review by droot — APPROVED** [review]

**Comment by phanimarupaka** [comment]
@mortent This change will format Kptfile in lexical order.

**Comment by mortent** [comment]
@phanimarupaka We should fix that. We need to have an openapi schema for the Kptfile anyway, so we should make sure that it is used here too. Created an issue for this: https://github.com/GoogleContainerTools/kpt/issues/2052

**Pull request merged**